### ISSUE TITLE
Issue moving from 1.5.3 to 1.6.0

### ISSUE THREAD

**Issue by rmarinho** [appium/appium-mac2-driver#209]
We have our CI running some tests on Mac with appium and this driver. CI machines are recreated for each run and started installing 1.6.0. We start getting this issue:

```
OpenQA.Selenium.WebDriverException : An unknown server-side error occurred while processing the command. Original error: Cannot read properties of undefined (reading 'info')
```
Here are some logs : [appium_mac-2.log](https://github.com/appium/appium-mac2-driver/files/11653288/appium_mac-2.log)

**Comment by mykola-mokhnach** [comment]
The patch has been published in 1.6.1

**Comment by rmarinho** [comment]
Thanks, fixed!
